### ISSUE TITLE
Independent authentication server for account

### ISSUE THREAD

**Issue by maboroshin** [yattee/yattee#751]
### Guidelines

- [X] I have searched the [issue tracker](https://github.com/yattee/yattee/issues) and I haven't found feature request like this

### Type

Improvement to existing feature

### Describe the feature

Each instance is prone to errors when connecting to YouTube. This is a recent trend. I would like to see separate accounts for Subscriptions and playlists. Piped already supports this.

* Instance for data sharing. For information not related to connection errors, such as subscriptions and playlists.
* Instance for accessing YouTube.

Errors are frequent. So, direct connection to YouTube may be an option. LibreTube has started this: [LibreTube 0.25.1](https://github.com/libre-tube/LibreTube/releases/tag/v0.25.1)

**Comment by stonerl** [comment]
The good news is Yattee already has the feature to separate the playback account from the login account.

The bad news is, this feature is currently broken 😞. Need to fix this.

We also have the option to directly connect to YouTube. Just disable the proxy in the location settings and Yattee will directly connect to the YouTube servers.

When you disable proxying in Yattee, it will then use conditional proxying. Mostly used for Music and Sports content.

That said, the problem isn't the way it connects. The recent outage of piped and invidious instances would not be prevented by these settings. The instances need to extract the correct url for the videos and all other metadata.

Some videos (music/sports) need to be accessed from the IP that requested it in the first place. E.g. the invidious server, that's why we have conditional proxying.

The URLs for videos usually are only usable for 24h and then need to be regenerated.

Libretube makes use of a feature that has been developed my the invidious devs to circumvent YouTube's latest restrictions.It is also included in the latest invidious release.

Libretube is a frontend for piped and basically they implement this feature to make piped usable again. But the piped server software should also receive this fix in the near future.

In a nutshell, we need to fix the account playback feature, but there is no reason to implement any measurements to circumvent YouTube blocking. That needs to be done by the server developers.

Yattee is a mere frontend for the servers and reimplementation of features that should be dealt with by the invidious/piped is a fruitless effort.

**Comment by maboroshin** [comment]
Thanks for the explanation. So it includes parts that can't be solved by direct connection.

I forgot to mention that I'm only using the release version on iOS. So, I had not seen the latest pre-release.

Invidious referred to this as a past issue as “under discussion”: https://github.com/iv-org/invidious/issues/4882
